README.md

```diff
@@ -28,71 +29,84 @@ There are many benefits in running your application as a unikernel: for more inf
 - 🧰 Go SDK for building unikernels programmatically; and
 - 🚀 _much more!_
 
+
 ## Installation
 
-You can quickly and easily install KraftKit using the interactive installer.  Simply run the following command to get started: 
+You can quickly and easily install KraftKit using the interactive installer.
+Simply run the following command to get started:
 
 ```shell
 curl --proto '=https' --tlsv1.2 -sSf https://get.kraftkit.sh | sh
 ```
 
 Alternatively, you can download the binaries from the [releases pages](https://github.com/unikraft/kraftkit/releases).
 
+See [additional installation instructions](https://unikraft.org/docs/cli/install).
+
+See also the [hacking documentation on how to build KraftKit from source](https://unikraft.org/docs/cli/hacking).
+
+
 ### Container build environment
 
-KraftKit ships a container build environment which you can use instead of installing any dependencies directly on your host.
-It includes the `kraft` binary as well as all the additional tools and libraries for building Unikraft unikernels.
-Simply attach a working directory on your host as a mount path volume mapped to `/workspace`, e.g.:
+KraftKit ships a container build environment which you can use instead of installing any dependencies directly on your host. It includes the `kraft` binary as well as all the additional tools and libraries for building Unikraft unikernels. Simply attach a working directory on your host as a mount path volume mapped to `/workspace`, e.g.:
 
 ```shell
 docker run -it --rm -v $(pwd):/workspace --entrypoint bash kraftkit.sh/base:latest
 ```
 
-The above command will drop you into a container shell.
-Simply type `exit` or Ctrl+D to quit.
+The above command will drop you into a container shell.  Simply type `exit` or Ctrl+D to quit.
 
-## Quickstart
-
-Building a unikernel with KraftKit is designed to be simple.
 
-Add a `Kraftfile` to your project directory, which specifies the libraries needed for your unikernel:
+## Quickstart
 
-```yaml
-specification: v0.5
+Running unikernels with `kraft` is designed to be simple and familiar.
+To test your installation of `kraft`, you can run the following:
 
-unikraft: stable
+```
+kraft run unikraft.org/helloworld:latest
+```
 
-libraries:
-  musl: stable
+Building unikernels is also designed to be simple.
+You can find some common project examples below:
 
-targets:
-  - name: default
-    architecture: x86_64
-    platform: qemu
-```
+| | Example |
+|-|:-|
+| ![](https://raw.githubusercontent.com/unikraft/catalog/main/icons/c.svg) | [Simple "Hello, world!" application written in C](https://github.com/unikraft/catalog/tree/main/examples/helloworld-c) |
```

Comment:
We need to have everything here with a cool table.

We can add another column which shows if last test/build passed with a checkmark or a `Build Passing` thing

Reply:
We can add that column to the catalog repo :)